tests/integration/targets/mssql_script/tasks/main.yml

```diff
@@ -1,9 +1,4 @@
 ---
-####################################################################
-# WARNING: These are designed specifically for Ansible tests       #
-# and should not be used as examples of how to write Ansible roles #
-####################################################################
-
```

Comment:
Why remove the above?

Reply:
I assumed it was only boilerplate from the template, but happy to roll those back in

Reply:
It's a generic boilerplate we add to all integration tests.

Reply:
Rolled back in